setup.cfg

```diff
@@ -43,7 +43,7 @@ dev =
     pytest-cov==2.10.1
     black==20.8b1
     isort==5.6.4
-    pylint==2.6.0
+    pylint>=2.10.0
```

Comment:
Current version is 2.17. We should use that

Reply:
2.17 actually causes a dependency issue with something else, iirc. I went back to a min version that worked and didn't show that collision

Reply:
If you could open a ticket about this, it would be good to track. No further update needed here then

Reply:
re-tested & saw no issue(s). updating.